README.md

```diff
@@ -0,0 +1,64 @@
+- Data layer - implements domain layer abstractions, the `DataSources`, that related to data persistence, REST calls, etc. 
+- Device layer - implements domain layer abstractions that are not related to data persistence and user interface and specific to android platform - android services, cloud messaging, so on
+- Presentation (UI) layer - all the functionality related to the Android user interface - activities, fragments, views
+
+<p align="center">
+  <img height="250" src="/images/arch.png" >
+</p>
+
+The core libraries for the layer's components communication are [**LiveData**](https://developer.android.com/topic/libraries/architecture/livedata), used to notify the UI changes and [**Coroutines**](https://kotlinlang.org/docs/reference/coroutines-overview.html) used to perform all background tasks.
+
+<p align="center">
+  <img src="/images/layers.png">
+</p>
+
+## Core Libraries
+The main libraries that we are using are:
+- [Android Architecture Components - Jetpack](https://developer.android.com/topic/libraries/architecture):
+ - [LiveData](https://developer.android.com/topic/libraries/architecture/livedata) which provides data objects that notify views when the underlying database changes.
+ - [ViewModel](https://developer.android.com/topic/libraries/architecture/viewmodel) which stores UI-related data that isn't destroyed on the app rotations.
+ - [Room](https://developer.android.com/topic/libraries/architecture/room), a SQLite object mapping library.
+ - [Android Navigation Component](https://developer.android.com/guide/navigation) used to navigate across different pieces of content within your app.
+- [Coroutines](https://kotlinlang.org/docs/reference/coroutines-overview.html) for asynchronous programming
+- [Coil](https://coil-kt.github.io/coil/), an image loading library for Android backed by Kotlin Coroutines.
+- [Koin](https://insert-koin.io/), a lightweight dependency injection framework for Kotlin.
+- [OkHttp](https://square.github.io/okhttp/) and [Retrofit](https://square.github.io/retrofit/) for network communication.
+- [Timber](https://github.com/JakeWharton/timber) one of the most popular logger on Android.
+- [Stetho](http://facebook.github.io/stetho/), a sophisticated debug bridge for Android applications.
+- [LeakCanary](https://square.github.io/leakcanary/), a memory leak detection library for Android.
+
+## Setup
+To set up a new project you have to follow these steps:
+- Clone the project
+- Update the `applicationId` in the app's build gradle file.
+- Change the package structure based on your application id.
+
+_We're currently working on a script to automatize this process._
+
+## Configuration and secrets
+The app's version name is defined in the project's [Gradle file](https://github.com/xmartlabs/GongAndroidBaseProject/blob/master/build.gradle). 
```

Comment:
I think you can use relative paths to point to code in the repo, it might be helpful in case the repo's name changes or you are looking at a diff branch

Reply:
```suggestion
The app's version name is defined in the project's [Gradle file](/build.gradle). 
```